src/%IPM/Main.cls

```diff
@@ -1429,7 +1528,7 @@ ClassMethod Load(ByRef pCommandInfo) [ Internal ]
 ClassMethod Install(ByRef pCommandInfo) [ Internal ]
 {
 	Set tRegistry = ""
-	Set tModuleName = $ZConvert($Get(pCommandInfo("parameters","module")), "L")
+	Set tModuleName = $Get(pCommandInfo("parameters","module"))
```

Comment:
Ah is this the fix for all module names showing up in lower case?

Reply:
Yes it is, and also in a few other places where "l" is explicitly set